docs/introduction/Car-registry-tutorial.rst

```diff
@@ -2,73 +2,76 @@
+
+(Not sure that "Sidechain" should be capitalized as often as it is, but left it in case it's an internal formatting choice.)
 
 User stories:
 #############
 
 1
-**Q: I want to add my car to a Car Registry Sidechain.**
+**Q: I want to add my car to the Car Registry Sidechain.**
 
-*A:* Create a new Car Entry Box, which contains car identification information (Unique car identifier, VIN, manufacturer, model, year, registration number), and certificate. Proposition in this box is my public key in this Sidechain. When I create a box, Sidechain should check car identification information and certificate to be unique in this Sidechain.
+*A:* Create a new Car Entry Box, which contains car identification information (VIN, manufacturer, model, year, registration number), and certificate. Proposition in this box is my public key in this Sidechain. When I create a box, the Sidechain should verify car identification information and certificate are unique in this Sidechain.
 
 2
-**Q: I want to create sell order to sell my car using Car Registry Sidechain.**
+**Q: I want to create a sell order to sell my car using Car Registry Sidechain.**
 
-*A:* I create a new Car Sell Order Box that contains the price in coins and information from the Car Entry Box. So cars can exist in the Sidechain as a Car Entry Box or as a Car Sell Order, but not at the same time. Also, this box contains the buyer’s public key. When I create a sell order, Sidechain should check if there is no other active sell order with this Car Entry Box. The current Sell Order consists of the same information that consists of the Car Entry Box plus description.
+*A:* I create a new Car Sell Order Box that contains the price in coins and information from the Car Entry Box. So cars can exist in the Sidechain either as a Car Entry Box or as a Car Sell Order, but not both at the same time. Also, this box contains the buyer’s public key. When I create a sell order, the Sidechain should verify that there is no other active sell order with this Car Entry Box. The current Sell Order consists of the same information that contained in the Car Entry Box plus description (is the description optional?).
 
 3
-**Q: I want to see all available Sell orders in Sidechain**
+**Q: I want to see all available Sell orders in the Sidechain.**
 
 *A:* Have additional storage, which is managed by ApplicationState and stores all Car Sell Orders. All these orders can be retrieved using the new HTTP API call. 
 
-
 4
 **Q: I want to accept a sell order and buy the car.**
 
-*A:* By accepting sell order, I create a new transaction in the Sidechain, which creates a new Car Entry Box with my public key as Proposition and transfers coins amount from me to the previous car owner.
+*A:* By accepting a sell order, I create a new transaction in the Sidechain, which creates a new Car Entry Box with my public key as Proposition and transfers the correct value of coins from me to the seller.
```

Comment:
*A:* By accepting a sell order, you create a new transaction in the sidechain, which creates a new Car Entry Box with your public key as the proposition and transfers the correct value of coins from you to the seller.

@zennermac